### ISSUE TITLE
[drop-down] Keyboard user is unable to interact with drop-down menu buttons

### ISSUE THREAD

**Issue by igpeev** [vmware/clarity#6568]
## Keyboard user is unable to interact with drop-down menu buttons
[StackBlitz](https://stackblitz.com/edit/clarity-dark-theme-v5-ehswjo?file=src/app/app.component.html)

## How to reproduce (use key-board only):
1.  Hit Tab until "ACTION BUTTON-S" becomes focused
2.  Expand Actions (hit Return/Enter)
3. Try to go down to actions AFTER Button 2 (using down keyboard arrow)
4. **The focus disappears and the keyboard user cannot focus back**

## Expected behaviour
The focus doesn't disappear.

## Additional notes
- When using `div`s instead of `button`s, the issue seems gone, BUT the disabled-looking dropdown-item is still clickable (as seen in the console), which is not desired behaviour.

## Versions
**Clarity project:**
- Clarity Angular/UI

**Clarity version:**
-  v5.x

**Framework:**
- Angular

**Framework version:**
Angular 11.2.14

**Device:**

- Type: [Mac]
- OS: [BigSur]
- Browser [Chrome]
- Version [96.0.4664.110]

**Comment by gnomeontherun** [comment]
@igpeev are these issues tracked by our internal a11y team? If so that is where we review and triage issues so we don't have duplicate issues. If they are not tracked by our a11y team, they should be. Adding @scroniser for visibility.
